### PR TITLE
Adding grid.py to lesson 2

### PR DESCRIPTION
**Pull request by mattclau**
Adding grid exercise for lesson 2.

### CODE REVIEW

**Comment by mattclau** [comment]
I'm pretty new to git.  I thought I needed to do a new pull request, but apparently it just puts my file in with my previous request until it is approved, so now fizzbuzz is there.

File: students/mattclau/lesson02/fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,101):
+    if i % 3 == 0 and i % 5 == 0:
+        print('FizzBuzz')
+    elif i % 3 == 0:
+        print('Fizz')
+    elif i % 5 == 0:
+        print('Buzz')
+    else:
+        print(i)
```
Comment (charlesrobison) [inline students/mattclau/lesson02/fizzbuzz.py]: Great!

File: students/mattclau/lesson02/grid.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+corner = '+'
+horz = '-'
+vert = '|'
+space = ' '
+
+def printHoriz(gridSize, sideLen):
```
Comment (charlesrobison) [inline students/mattclau/lesson02/grid.py]: Per PEP8 function names and variables should be snake_case.  def print_horiz(grid_size, side_len):

File: students/mattclau/lesson02/grid.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    for i in range(gridSize):
+        printHoriz(gridSize, SideLen)
+        printVert(gridSize, SideLen)
+    printHoriz(gridSize, SideLen)
+
+side = input("Length of sides: ")
+gridSize = input("Size of grid: ")
+
+
+printGrid(int(gridSize), int(side))
```
Comment (charlesrobison) [inline students/mattclau/lesson02/grid.py]: Works well!

**Pull request merged**